### ISSUE TITLE
HL7 Field Preview or Reference in Editor

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#4280]
Would be really nice to be able to somehow preview the hl7 segment,element, component, subcomponent, definitions in the code. Personally, I see this as an intellisense thing where I could type "['PID']." and have a list of the components within it with the field name appended (e.g. on each line: ['PID.3'] -- External Patient ID, ['PID.4'] -- Alternate Patient ID , ['PID.5'] -- Patient Name)

Another example I've seen of this is Iguana is have the intellisense and a line breakdown of the compilation with this description to the right. Doesn't look like i can attach images, but I can send an example if you'd like.

The bottom line is having a quick an easy reference for hl7 segment definitions be accessible to make writing mapping code much easier.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-4404
Reporter: pdevito
Created: 2019-04-25T08:55:28.000-0700

**Comment by rbeckman-nextgen** [comment]
added an example image

Imported Comment. Original Details:
Author: pdevito
Created: 2019-04-25T08:56:36.000-0700

**Comment by rbeckman-nextgen** [comment]
The editor example looks very nice. It would be nice if we could do the same in Mirth ie msg.PID[3][1][1] vs msg['PID']['PID.3']['PID.3.1']['PID.3.1.1']

Imported Comment. Original Details:
Author: michaelleehobbs
Created: 2019-11-07T14:21:58.000-0800
